releases/release-1.30/release-notes/maps/pr-122612-map.yaml

```diff
@@ -1,5 +1,5 @@
 pr: 122612
 releasenote:
   text: |
-    Removed the `SecurityContextDeny` admission plugin, which was deprecated since `v1.27`. The Pod Security Admission plugin, available since v1.25, is now recommended instead. Refer to https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#securitycontextdeny for more information.
+    The deprecated `SecurityContextDeny` admission plugin, which has been deprecated since `v1.27`, has been removed. It is recommended to use the Pod Security Admission plugin instead, which has been available since v1.25. Refer to the [Kubernetes documentation](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#securitycontextdeny) for more information.
```

Comment:
Can you update this Release Note with the latest change shown here? https://github.com/kubernetes/kubernetes/pull/122612#issuecomment-2054117692. Thanks!